docs/modules/operation/pages/provisioning/detectors/HostResourceSWRunDetector.adoc

```diff
@@ -8,14 +8,14 @@ Most modern _SNMP agents_, e.g. _Net-SNMP agent_ and the _Microsoft SNMP service
 
 NOTE: This detector implements the configuration parameters inherited from the <<provisioning/detectors/SnmpDetector.adoc>>.
 
-==== Detector facts
+== Detector facts
```

Comment:
These changes have already been pushed to foundation-2021.  Will that conflict with this commit?